www/vorj00/project_01/contact.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="description" content="Berkshire Hathaway official website">
+    <meta name="keywords" content="Berkshire Hathaway,investing,website,webpage,official">
+    <meta name="author" content="Jakub Voracek">
+    <title>Contact | Berkshire Hathaway</title>
+    <link rel="shortcut icon" href="assets/img/favicon.ico">
+    <link rel="icon" type="image/png" sizes="192x192" href="assets/img/favicon.png">
+    <link rel="icon" type="image/x-icon" href="assets/img/favicon.svg">
+    <link rel="stylesheet" href="assets/css/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <a href="index.html">
+            <picture class="header__logo">
+                <source type="image/svg+xml" srcset="assets/img/logo.svg">
+                <img src="assets/img/logo.png" alt="Berkshire Hathaway">
+            </picture>
+        </a>
+        <nav class="nav">
+            <a href="subsidies.html" class="nav__item">Subsidiary companies</a>
+            <a href="reports.html" class="nav__item">Reports</a>
+            <a href="news.html" class="nav__item">News</a>
+            <a href="contact.html" class="nav__item nav__item--active">Contact</a>
+        </nav>
+    </header>
+    <main class="main">
+        <h1>Contact</h1>
```

Comment:
co treba to trochu ozivit ikonama, treba pomoci fontawesome?

Reply:
nevěděl jsem právě, jestli můžeme používat, ale zrovna tady nevím, jestli by to bylo estetický